ladybug/epw.py

```diff
@@ -1,7 +1,10 @@
 # coding=utf-8
+from __future__ import division
+
 from .location import Location
 from .analysisperiod import AnalysisPeriod
-from .datatype import DataPoint  # Temperature, RelativeHumidity, Radiation, Illuminance
+from .datatype import DataPoint
+from .datatypenew import DataTypes
```

Comment:
Before I continue with this PR let's merge `datatype` and `datatypenew` modules and we discussed before: https://github.com/ladybug-tools/ladybug/pull/96#issuecomment-449221652

Reply:
This has been done now.  Please proceed.